control-plane/roles/logging/README.md

```diff
@@ -0,0 +1,3 @@
+# logging
+
+TODO: required variable: logging_loki_image_version
```

Comment:
Please provide a role documentation similar to the other roles of this repository.

Reply:
https://github.com/metal-stack/metal-roles/commit/0db3c2c09d2b9459b70366b23fde0bcb212f085e
https://github.com/metal-stack/metal-roles/commit/791d69854e20dedadde7f5b78a02f3524656dea5